README.md

```diff
@@ -31,7 +31,10 @@ ansible-galaxy install -r requirements.yml -p roles/
 
 Role Variables
 --------------
-
+public_keys_path: path for public keys. Default: public_keys
+public_all_filekeys_path: temporary path for work with public keys. Default : /tmp/all_public_keys
+sshlog_enable: for enable sshlog . Default: 0
+authorized_key_exclusive: exclusive public keys of publick_keys_path. Default: yes
```

Comment:
extra `k` in  `publick_keys_path`, also I would use backticks to highlight this config options

Reply:
Fixed the extra k in public and put the backticks for the new vars in readme file.